aspnetcore/mvc/views/tag-helpers/builtin.md

```diff
@@ -11,7 +11,7 @@ Microsoft has included 17 Tag Helpers that can be immediately used in .net Core
 
 **[AnchorTagHelper](builtin/resources/AnchorTagHelper.md)**
 
-[comment]: **[CacheTagHelper](builtin/resources/AnchorTagHelper.md)**
+**[CacheTagHelper](builtin/resources/CacheTagHelper.md)**
```

Comment:
I would suggest leaving this as is because this is a list of Core Tag Helpers and their actual names. That is, this is not about in general "Cache" Tag Helpers because that might imply it covers the distributed cache tag helper which it does not.  Happy to change, just IMHO.